repository/src/main/resources/alfresco/repository.properties

```diff
@@ -1202,7 +1202,7 @@ repo.event2.filter.nodeAspects=sys:*
 repo.event2.filter.childAssocTypes=rn:rendition
 # Comma separated list of users which should be excluded
 # Note: username's case-sensitivity depends on the {user.name.caseSensitive} setting
-repo.event2.filter.users=System, null
+repo.event2.filter.users=

```

Comment:
I didn't work out why the user sending making the changes shows up as "null" - I could work around this by using a "run as system", but I figured it's safer to avoid filtering any events by default so that Elasticsearch has the potential to receive any updates to metadata/permissions.